src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
```

Comment:
The path to the image in the `background-image` property is absolute. It's better to use relative paths in your code. This will make your code more portable, as the images will still be found even if the location of the CSS file changes.